src/v2/Apps/Order/Components/CreditCardInput.tsx

```diff
@@ -1,90 +1,103 @@
-import { BorderBox, Sans, color, themeProps } from "@artsy/palette"
-import { fontFamily } from "@artsy/palette/dist/platform/fonts"
-import {
-  BorderProps as InputBorderProps,
-  borderMixin as inputBorder,
-} from "v2/Components/Mixins"
+import { BorderBox, themeProps, useThemeConfig, Text } from "@artsy/palette"
+import { borderMixin, v3BorderMixin } from "v2/Components/Mixins"
 import React from "react"
 import type {
   StripeCardNumberElementChangeEvent,
   StripeError,
 } from "@stripe/stripe-js"
 import { CardElement } from "@stripe/react-stripe-js"
 import styled from "styled-components"
+import { useState } from "react"
+import { fontFamily } from "@artsy/palette/dist/platform/fonts"
 
-export const StyledCardElement = styled(CardElement)`
+export const StyledCardElement = styled(CardElement)<{ isV3?: boolean }>`
   width: 100%;
-  padding: 9px 10px;
+  ${props => (props.isV3 ? "padding: 12px 10px" : "padding: 9px 10px")}
 `
 
 // Re-uses old input border behavior
-const StyledBorderBox = styled(BorderBox)<InputBorderProps>`
-  ${inputBorder};
+export interface BorderProps {
+  hasError?: boolean
+  isV3?: boolean
+}
+
+const StyledBorderBox = styled(BorderBox)<BorderProps>`
   padding: 0;
   height: 40px;
+  ${props => (props.isV3 ? v3BorderMixin : borderMixin)}
+  ${props => props.isV3 && "margin-bottom: 20px"}
```

Comment:
You should use [`getThemeConfig`](https://github.com/artsy/palette/blob/5a263d623b5eb27f6427489db05101af2285cadd/packages/palette/src/Theme.tsx#L72) instead here.

Reply:
Is there an example of how to use `getThemeConfig`? I don't see it being used anywhere in force.

Reply:
It's used [frequently in Palette](https://github.com/artsy/palette/search?q=getthemeconfig) — I guess we haven't had much of a need for it in Force. But it's basically the exact same thing as `useThemeConfig` but with a slightly different call signature: the first argument is the props, second argument is the v2/v3 configuration.